src/Support/Http/Middleware/AttemptAuthentication.php

```diff
@@ -40,7 +40,7 @@ public function handle($request, Closure $next, ...$guards)
     protected function attemptAuthentication(array $guards): void
     {
         if (empty($guards)) {
-            $guards = [null];
+            $guards = [config('lighthouse.guard', null)];
```

Comment:
```suggestion
            $guards = (array)config('lighthouse.guard', []);
```

This would allow passing either an array or string to the `lighthouse.guard` config to still keep the option for multiple default guards (think `['api', 'web']` for example).

Reply:
An empty array is not a good default here, it would not check any guards in that case.

Reply:
You are correct it should be `[null]` as default indeed, I missed that.